src/components/Dropdowns/ColorRampDropdown.tsx

```diff
@@ -61,6 +66,9 @@ const ColorRampSelector: React.FC<ColorRampSelectorProps> = (propsInput): ReactE
   const props = { ...defaultProps, ...propsInput } as Required<ColorRampSelectorProps>;
   const theme = useContext(AppThemeContext);
 
+  const colorRampsToDisplay = props.colorRampsToDisplay;
+  const categoricalPalettesToDisplay = props.categoricalPalettesToDisplay;
```

Comment:
another tiny annoying suggestion: could be one destructuring assignment, like so: `const { colorRampsToDisplay, categoricalPalettesToDisplay } = props;`